qa/src/test/resources/kapua-datastore-transport-client-setting.properties

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Eurotech - initial API and implementation
+#
+###############################################################################
+
+#
+# Elasticsearch Properties
+#NOTE: the datastore client provider implementation must provide:
+#      - a static init method (that initializes the client)
+#      - a static getInstance method (that returns the instance)
+datastore.elasticsearch.client.provider=org.eclipse.kapua.service.datastore.client.embedded.EsEmbeddedClientProvider
+#datastore.elasticsearch.client.provider=org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider
+
+#timeouts
+datastore.query.timeout=60000
+datastore.scroll.timeout=60000
+
+datastore.elasticsearch.cluster=kapua-datastore
+datastore.elasticsearch.node=192.168.33.10
```

Comment:
Well I suggest to use some mechanism which can be run locally, on travis, husdon and jenkins.

If that currently works then there is no change required.

If it isn't required, it could be removed, right?